### ISSUE TITLE
Import breaks with JAX version 0.4.24 due to removed function

### ISSUE THREAD

**Issue by oraisa** [DPBayes/jax-chacha-prng#25]
Error message from importing `chacha.random`:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/oraisa/Library/Caches/pypoetry/virtualenvs/multiple-syn-datasets-ml-CFH753D6-py3.10/lib/python3.10/site-packages/chacha/random.py", line 28, in <module>
    import chacha.cipher as cc
  File "/Users/oraisa/Library/Caches/pypoetry/virtualenvs/multiple-syn-datasets-ml-CFH753D6-py3.10/lib/python3.10/site-packages/chacha/cipher.py", line 19, in <module>
    from chacha.jax_ops import chacha20_block as _block
  File "/Users/oraisa/Library/Caches/pypoetry/virtualenvs/multiple-syn-datasets-ml-CFH753D6-py3.10/lib/python3.10/site-packages/chacha/jax_ops.py", line 48, in <module>
    xla_client.register_cpu_custom_call_target(
AttributeError: module 'jaxlib.xla_client' has no attribute 'register_cpu_custom_call_target'. Did you mean: 'register_custom_call_target'?
```

**Comment by lumip** [comment]
Fixing this properly will take a bit more work but now there's at least a hotfix patch release (1.4.2) that installs only compatible jax versions.

**Comment by lumip** [comment]
fixed in 1.4.3
